src/Containers/urls.js

```diff
@@ -12,7 +12,7 @@ export async function getFinalUrl(url, accessToken) {
   switch (u.host.toLowerCase()) {
     case 'github.com':
       if (!accessToken) {
-        const proxyUrl = new URL(process.env.RAW_GIT_PROXY_URL_NEW)
+        const proxyUrl = new URL(process.env.RAW_GIT_PROXY_URL)
```

Comment:
This looks wrong.. no accessToken should use the new /model base on proxy url i think